client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.controller.js

```diff
@@ -0,0 +1,85 @@
+import { COMFORT_EXCHANGE_TYPE_ERROR } from './pack-xdsl-access-comfort-exchange.constant';
+
+angular.module('managerApp').controller(
+  'XdslAccessComfortExchangeCtrl',
+  class XdslAccessComfortExchangeCtrl {
```

Comment:
You can only export the `class` and then import it from your `*.module.js` definition (like the state definition).

Reply:
Sorry I don't understand what you want I'm doing on this.

Reply:
@antleblanc suggest you to not directly declare your controller through `angular.module("managerApp")` with the class definition.

Instead just`export default class yourClass`, and then in the `module.js` file. import the exported controller, to declare it in the module.